src/plot-data.ts

```diff
@@ -1238,7 +1237,8 @@ export abstract class PlotData {
       let selected = true
       if (this.vertical) { var seg_list = this.vertical_axis_coords[i]; } else { var seg_list = this.horizontal_axis_coords[i]; }
       for (let j=0; j<this.axis_list.length; j++) {
-        var inside_band = this.is_inside_band(seg_list[j][0], seg_list[j][1], j);
+        // var inside_band = this.is_inside_band(seg_list[j][0], seg_list[j][1], j);
```

Comment:
If we don't need these commented statement anymore, we can just remove them to prevent them from polluting the code

Reply:
Removed